pkgs/top-level/all-packages.nix

```diff
@@ -15252,7 +15252,9 @@ in
 
   shaarli-material = callPackage ../servers/web-apps/shaarli/material-theme.nix { };
 
-  matomo = callPackage ../servers/web-apps/matomo { };
+  inherit (callPackage ../servers/web-apps/matomo {})
```

Comment:
`callPackage` -> `callPackages`

Reply:
What's the difference? IIRC it seemed kind of arbitrary which one people used for multiple packages.